src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  align-content: center;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__logo-link:link,
+.header__logo-link:hover,
+.header__logo-link:active,
+.header__logo-link:visited {
+  height: 40px;
+  border: none;
+  cursor: pointer;
```

Comment:
`<a>` element doesn't have any borders by default and the cursor is pointer. Redundant.

Reply:
removed the redundant code